libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -163,6 +164,7 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
    * useful when the data source does not live in the source code. If the
    * search requires calling a remote data source, use `searchAsync` instead of
    * `search`.
+   * @deprecated Use `searchAsync` instead.
```

Comment:
```suggestion
   * @deprecated We recommend against using this property. To call a remote data source, use the `searchAsync` event instead.
```
Mostly the same changes as the first deprecation note, but I'd also pull in content from the last sentence of the description to start the second sentence with a "to" clause explaining the use case. And I'd delete that last sentence in the description.